tuples.md

```diff
@@ -0,0 +1,67 @@
+# Writing Assignment - Tuples
+
+So far you have seen how to use arrays for holding a set of values and then iterate through them, for example:
+
+```swift
+var distances: [Int] = [10, 15, 20]
+
+for distance in distances {
+  print(distance)
+}
+```
+
+Arrays are a useful method for collecting variables, but limited as the values they contain can only be of one type. In this exercise we will introduce another, more flexible content type, the tuple.
+
+A tuple is a flexible variable type that allows you to group multiple variables into one convenient compound variable. The variables contained within a tuple can be of any type (for example a string, integer or even arrays) and you can mix the types. This makes tuples useful for creating a container variable for related values, for example:
+
+```swift
+let completedDistance = ("km", 10, true)
+```
+
+This tuple represents an entry from a fitness tracker, recording a distance unit as a string, a distance as an integer and a boolean to show completion.
+
```

Comment:
great example showing multiple types. It's great that you encase this in a real world example, and this might be slight, but move that up before you show the code. Start with, "For example what if I wanted to show an entry from my  fitness tracker that contained the length of my race, the units it was in, and whether I had completed the race or not by using a boolean.

Or for example, going back to our movie example, we can say "For example, what if I wanted to show not only my favorite movie and director but also the year that it was made. That would look like this.
(show code example)